ci/run_envoy_docker.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+[[ -z "${IMAGE_NAME}" ]] && IMAGE_NAME="lyft/envoy-build-ubuntu"
+[[ -z "${IMAGE_ID}" ]] && IMAGE_ID="latest"
+[[ -z "${ENVOY_API_DOCKER_BUILD_DIR}" ]] && ENVOY_API_DOCKER_BUILD_DIR=/tmp/envoy-api-docker-build
+
+mkdir -p "${ENVOY_API_DOCKER_BUILD_DIR}"
+docker run -t -i -u $(id -u):$(id -g) -v "${ENVOY_API_DOCKER_BUILD_DIR}":/build \
+  -v "$PWD":/source "${IMAGE_NAME}":"${IMAGE_ID}" /bin/bash -c "cd source && $*"
```

Comment:
We don't need this script right? because we are building in circle ci?

Reply:
It's useful for developers; I use it when generating the docs in my local flow, e.g. `run_envoy_docker.sh './ci/do_ci.sh bazel.docs'`.